lua/ulx/modules/sh/cfc_curse.lua

```diff
@@ -72,6 +154,26 @@ function cmd.cursePlayers( callingPlayer, targetPlayers, effectName, durationMin
         return
     end
 
+    if smallestAmount == largestAmount and smallestAmount == 1 then
+        amount = false
+    end
+
+    if amount then
+        local amountStr
+
+        if smallestAmount == largestAmount then
+            amountStr = tostring( smallestAmount )
+        else
+            amountStr = smallestAmount .. " to " .. largestAmount
+        end
+
+        local durationAppend, briefly = getDurationStrings( durationSeconds, false )
+
+        ulx.fancyLogAdmin( callingPlayer, isSilent, "#A" .. briefly .. " cursed #T with " .. amountStr .. " random effects" .. durationAppend, targetPlayers )
+
+        return
+    end
```

Comment:
Printout for amount-based curses. Notes the min/max amount of received effects for any given player, in case effect incompatibilities caused some players to receive different amounts.